playbooks/common/openshift-cluster/redeploy-certificates/ca.yml

```diff
@@ -294,7 +294,7 @@
       client_path: "{{ openshift.common.config_base }}/master/admin.kubeconfig"
       ca_path: "{{ openshift.common.config_base }}/master/ca-bundle.crt"
   - name: Lookup default group for ansible_ssh_user
-    command: "/usr/bin/id -g {{ ansible_ssh_user }}"
+    command: "/usr/bin/id -g {{ ansible_ssh_user | quote }}"
```

Comment:
ok, testing this. Do you know by chance if that pwd.getpwnam() which user module uses works with domain username format? ie: 'domain\user' The customer had confirmed that quoting the command worked so I was going with that. I can't come up with a good test case.
```
>>> import pwd
>>> pwd.getpwnam('dom\user')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'getpwnam(): name not found: dom\\user'
```